infrastructure/ansible/roles/traffic-monitor/defaults/main.yml

```diff
@@ -54,3 +55,11 @@ tm_traffic_ops_url: 'https://localhost'
 tm_traffic_ops_allow_insecure: true
 tm_traffic_ops_target_cdn: 'some-cdn'
 tm_traffic_ops_http_listener: ':80'
+
+# traffic_ops.cfg https listener configuration
+tm_traffic_ops_use_https_listener: False
+tm_traffic_ops_https_listener: ':443'
+tm_traffic_ops_key_file_location: "{{ tm_conf_dir }}/traffic_monitor_ssl.key"
+tm_traffic_ops_cert_file_location: "{{ tm_conf_dir }}/traffic_monitor_ssl.crt"
+tm_traffic_ops_https_cert: ""
+tm_traffic_ops_https_key: ""
```

Comment:
I don't think it's in the best interests of OSS contributors to dictate how key material is distributed.  Instead, it would be appropriate to assert it's existence however.